examples/tutorials/contour-map.py

```diff
@@ -0,0 +1,104 @@
+"""
+Creating a map with contour lines
+=================================
+
+Plotting a contour map is handled by :meth:`pygmt.Figure.grdcontour`.
+"""
+
+import pygmt
+
+# Load sample earth relief data
+grid = pygmt.datasets.load_earth_relief(resolution="05m", region=[-92.5, -82.5, -3, 7])
+
+########################################################################################
+# Create contour plot
+# -------------------
+#
+# The :meth:`pygmt.Figure.grdcontour` method takes the grid input.
+# It plots annotated contour lines, which are thicker and have the
+# elevation/depth written on them, and unannotated contour lines.
+# In the example below, the default contour line intervals are 500 meters,
+# with an annotated contour line every 1000 meters.
+# By default, it plots the map with the
+# equidistant cylindrical projection and with no frame.
+
+fig = pygmt.Figure()
+fig.grdcontour(grid=grid)
+fig.show()
+
+########################################################################################
+# Contour line settings
+# ---------------------
+#
+# Use the ``annotation`` and ``interval`` arguments to adjust contour line intervals.
+# In the example below, there are contour intervals every 250 meters and
+# annotated contour lines every 1,000 meters.
+
+fig = pygmt.Figure()
+fig.grdcontour(
+    annotation=1000,
+    interval=250,
+    grid=grid,
+)
+fig.show()
+
+########################################################################################
+# Contour limits
+# --------------
+#
+# The ``limit`` argument sets the minimum and maximum values for the contour lines.
+# The argument takes the low and high values,
+# and is either a list (as below) or a string ``limit="-4000/0"``.
+
+fig = pygmt.Figure()
+fig.grdcontour(
+    annotation=1000,
+    interval=250,
+    grid=grid,
+    limit=[-4000, 0],
```

Comment:
It seems that the elevation/depth of this region (`[-92.5, -82.5, -3, 7]`) are all in the limit of -4000 to 0, so the `limit` argument makes no difference. Maybe it's better to change the `limit` to other values, e.g., `[-4000, -2000]`.

Reply:
I made this change to lower the upper limit